src/test/java/seedu/address/model/affiliation/AffiliationModifierTest.java

```diff
@@ -78,4 +78,23 @@ public void nameChangeAffiliation_validParam_success() {
                 patientB.getName(), newName, model);
         assertTrue(doctorA.getAffiliations().contains(new Affiliation(newName.fullName)));
     }
+
+    @Test
+    public void nameChangeAffiliationHistory_validParam_success() {
+        Person doctorAlice = new DoctorBuilder().withName("Alice")
+            .withAffiliationHistory("Bob").build();
```

Comment:
Maybe to increase test coverage, you can add another `AffiliationHistory` here, say "Thomas", for Alice? i.e. someone that is not in the model.

Reply:
Added, thaanks!